stripe/res/values-night/colors.xml

```diff
@@ -16,12 +16,11 @@
     <color name="stripe_paymentsheet_textinputlayout_hint">#61FFFFFF</color>
     <color name="stripe_paymentsheet_textinput_color">#DEFFFFFF</color>
     <color name="stripe_paymentsheet_payment_option_unselected_stroke">#1FFFFFFF</color>
-    <color name="stripe_paymentsheet_payment_method_label_text">#ffffff</color>
-    <color name="stripe_paymentsheet_toolbar_button_background">#1FFFFFFF</color>
-    <color name="stripe_paymentsheet_form_background_default">#1Fffffff</color>
-    <color name="stripe_paymentsheet_form_background_disabled">#1Fffffff</color>
+    <color name="stripe_paymentsheet_payment_method_label_text">#DEFFFFFF</color>
+    <color name="stripe_paymentsheet_payment_method_label_text_disabled">#61FFFFFF</color>
+    <color name="stripe_paymentsheet_elements_background_default">#474747</color>
```

Comment:
Just curious what does elements mean to you?

Reply:
Form and CardView. Elements like Stripe Elements, agree it's very generic. Open to a better name.